test/Driver/linker-args-order-linux.swift

```diff
@@ -0,0 +1,11 @@
+// Statically link a "hello world" program
+// REQUIRES: OS=linux-gnu
+// REQUIRES: static_stdlib
+print("hello world!")
+// RUN: %empty-directory(%t)
+// RUN: %target-swiftc_driver -v -static-stdlib -o %t/static-stdlib %s -Xlinker --no-allow-multiple-definition 2>&1| %FileCheck %s
```

Comment:
GenericUnix is still used in other places, and it's useful to be able to run the test even on Darwin. (Also, not actually compiling and linking would run faster.)